src/constants/index.js

```diff
@@ -2,11 +2,15 @@
 
 const { stripIndents } = require('common-tags');
 
-const DEFAULT_HEADING = `🚀 *Deployed to production!* Related stories:\n _PRs need correct syntax to appear in the list. Problem? Please report it <https://github.com/target/captains-log/issues|here>._
+const DEFAULT_HEADING = (
+  strings,
+  owner,
+  repo,
+) => `🚀 *${owner}/${repo} deployed to production!* Related stories:\n _PRs need correct syntax to appear in the list. Problem? Please report it <https://github.com/target/captains-log/issues|here>._
```

Comment:
Do we expect to potentially use `strings` in the future? If not, I'd say this could be a plain old helper function. But template literal works too.

Reply:
I agree, it is probably overkill. I kept it a template literal initially just to indicate that this is used for ALL messages. It is effectively a function now, and I think it could be modified to be better, so I may just leave it. I think you've got a good point though, it really should be a helper function until it doesn't need to be.